once/docker-entrypoint.sh

```diff
@@ -385,7 +385,65 @@ secrets:
 EOF
 }
 
+docker_authgear_create_project_accounts() {
+	docker_postgresql_temp_server_start
+
+	init_input="$(mktemp)"
+	{
+		echo 'accounts'
+		echo "$AUTHGEAR_HTTP_ORIGIN_ACCOUNTS"
+		echo "$AUTHGEAR_HTTP_ORIGIN_PORTAL"
+		echo 'sms'
+		echo 'y'
+		echo 'postgresql'
+	} >> "$init_input"
+	init_output="$(mktemp -d)"
+	authgear init --for-helm-chart <"$init_input" -o "$init_output"
+	# Override authgear.yaml
+	# FIXME: Enable public signup.
+	cat > "$init_output"/authgear.yaml <<EOF
+authenticator:
+  oob_otp:
+    sms:
+      phone_otp_mode: sms
+http:
+  public_origin: "$AUTHGEAR_HTTP_ORIGIN_ACCOUNTS"
+id: accounts
+oauth:
+  clients:
+  - client_id: portal
+    issue_jwt_access_token: true
+    name: Portal
+    post_logout_redirect_uris:
+    - "${AUTHGEAR_HTTP_ORIGIN_PORTAL}/"
+    redirect_uris:
+    - "${AUTHGEAR_HTTP_ORIGIN_PORTAL}/oauth-redirect"
+    x_application_type: traditional_webapp
+search:
+  implementation: postgresql
+ui:
+  signup_login_flow_enabled: true
+verification:
+  claims:
+    email:
+      enabled: false
+      required: false
```

Comment:
Feel like we need a go cmd program to do this, or we might forget to update this config in the future.

Reply:
We do have such command `authgear init --for-helm-chart`, but it is currently interactive-only, thus not suitable for scripting purpose. We can enhance it to make it detect the stdin is tty, and enter interactive / non-interactive mode automatically. Then we need to turn each question into a flag. If the flag is supplied, then the interactive question is skipped. In non-interactive mode, a missing flag means use the default.

I did not do that because it would take me some time to implement the above.